.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and Deploy Docs
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    runs-on: ubuntu-20.04
```

Comment:
why not using ubuntu lts 22.04?

Reply:
I copied it from the previous/current implementation: https://github.com/SAP/ui5-tooling/blob/v2/.github/workflows/deploy-docs.yml#L9

To be honest, didn't pay much attention to that.
Do you think we should update it? Is this 22.04 some LTS?

Reply:
keep it... will update the ubuntu version in a next step